### PR TITLE
[MAISTRA-430] Make operator print the correct version fields

### PR DESCRIPTION
**Pull request by jwendell**

### CODE REVIEW

**Review by brian-avery — APPROVED** [review]

**Review by dmitri-d — APPROVED** [review]

File: istio-operator/istio-operator.spec [hunk 1]
```diff
@@ -78,7 +78,7 @@ tar zxf %{SOURCE1} -C OPERATOR/src/github.com/maistra/istio --strip=1
 cd OPERATOR
 export GOPATH=$(pwd):%{gopath}
 pushd src/github.com/maistra/istio-operator/
-./tmp/build/build.sh
+VERSION=%{version} GITREVISION=%{git_shortcommit} GITSTATUS=Clean GITTAG=%{version}-%{release} ./tmp/build/build.sh
```
Comment (brian-avery) [inline istio-operator/istio-operator.spec]: GITTAG doesn't seem related to a git tag. And git describe outputs something different than this.

Comment (jwendell) [reply istio-operator/istio-operator.spec]: I can change `VERSION` to print `%{version}-%{release}` (something like `1.0.0-1.el8`).
Suggestions for `GITTAG`? Its use is primarily for developers, not sure if it's useful here. Could put the version again?

Comment (brian-avery) [reply istio-operator/istio-operator.spec]: Yeah, my thought is Git tag isn't useful unless it's something that can be found as a Git tag. So I'd say if you were going to put anything here it would be version since that's what will be used as the tag.

Comment (brian-avery) [reply istio-operator/istio-operator.spec]: i.e. Tag has a special meaning in reference to Git and that's likely what users would try to interpret it as.

Comment (jwendell) [reply istio-operator/istio-operator.spec]: Sounds good. I made the changes, swapped them.
Built an rpm and here's the output:
```
# istio-operator -version
version.BuildInfo{Version:"1.0.0-1.el8", GitRevision:"9af4b83", BuildStatus:"Clean", GitTag:"1.0.0", GoVersion:"go1.12.7", GoArch:"linux/amd64", OperatorSDK:"v0.2.1"}
```

Comment (brian-avery) [reply istio-operator/istio-operator.spec]: LGTM. Thanks.
